plugins/plugin-chart-echarts/src/utils/series.ts

```diff
@@ -39,7 +44,43 @@ export function extractTimeseriesSeries(
     }));
 }
 
-export function extractGroupbyLabel(datum: DataRecord, groupby: string[]): string {
+export function formatSeriesName({
```

Comment:
Can we change the signature to `formatSeriesName(name, { numberFormatter, timeFormatter })` and make `name` an required argument?

Reply:
Thanks, I wasn't aware of this being a common pattern, I'll update accordingly 👍